Sources/AblyChat/EmitsDiscontinuities.swift

```diff
@@ -1,5 +1,14 @@
 import Ably
 
+public struct DiscontinuityEvent: Sendable, Equatable {
+    /// The error, if any, associated with this discontinuity.
+    public var error: ARTErrorInfo?
```

Comment:
What a discontinuity without an error?

Reply:
I assume that's "what _is_ a discontinuity without an error"? I think that 30cfd30c5dcc74951d39de454f2c0a4ef9e5a56a explained this?